src/cbmc/cbmc_parse_options.cpp

```diff
@@ -602,6 +602,7 @@ int cbmc_parse_optionst::get_goto_program(
       }
 
       languaget *language=get_language_from_filename(filename);
+      language->get_language_options(cmdline);
```

Comment:
This is potentially also missing from `clobber_parse_options.cpp` and possibly other places (by looking for references to `get_language_from_filename` and `get_language_from_mode`). Is there a way we can make this less error prone, or at least add a clear error message if `get_language_options` hasn't been called for an instance of `language`